app/uk/gov/hmrc/customs/emailfrontend/connectors/UpdateVerifiedEmailConnector.scala

```diff
@@ -40,11 +41,12 @@ class UpdateVerifiedEmailConnector @Inject()(appConfig: AppConfig,
 
     auditRequest(currentEmail, newEmail, eori, "changeEmailAddressVerified")
 
-    http.PUT[VerifiedEmailRequest, VerifiedEmailResponse](appConfig.updateVerifiedEmailUrl, request).map { resp =>
-      auditRequest(currentEmail, newEmail, eori, "changeEmailAddressConfirmed")
-      Right(resp)
-
-    } recover {
+    http.put(url"${appConfig.updateVerifiedEmailUrl}")
+      .withBody[VerifiedEmailRequest](request)
+      .execute[VerifiedEmailResponse].map { resp =>
+        auditRequest(currentEmail, newEmail, eori, "changeEmailAddressConfirmed")
```

Comment:
Please format it from line 47-49 as it is not intended correctly

Reply:
even after formatting in intellij, it looks the same.. so leaving it unfixed